angular/projects/spark-angular/src/lib/directives/sprk-centered-column/sprk-centered-column.directive.ts

```diff
@@ -0,0 +1,14 @@
+import { Directive, ElementRef, HostBinding } from '@angular/core';
+
+@Directive({
+  selector: '[sprkCenteredColumn]'
+})
+export class SprkCenteredColumnDirective {
+
+  @HostBinding('class.sprk-o-CenteredColumn') true;
+
+  /**
+   * @ignore
+   */
+  constructor(public ref: ElementRef) {}
```

Comment:
We don't need the ElementRef import and injection into the constructor since we are not using it.